core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -77,25 +73,26 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.enableCorrectnessFeedback = async function() {
-    expect(await enableCorrectnessFeedbackButton.isDisplayed()).toBe(true);
     await action.click(
-      'Enable Correctness Feedback Button', enableCorrectnessFeedbackButton);
+      'Enable Correctness Feedback button', enableCorrectnessFeedbackButton);
   };
 
   this.expectAvailableFirstStatesToBe = async function(names) {
     var options = await initialStateSelect.all(by.tagName('option'))
       .map(async function(elem) {
+        await waitFor.visibilityOf(elem, 'Element taking too long to appear');
```

Comment:
Can you make the error message more descriptive by saying which element is taking too long to appear?

Reply:
Sure! Done! :)

Reply:
Done